crates/proof-of-sql-parser/src/identifier.rs

```diff
@@ -85,6 +85,7 @@ impl AsRef<str> for Identifier {
     }
 }
 
+#[allow(clippy::missing_panics_doc)]
```

Comment:
Rather than putting these on each test module, it should work to put
```rust
#![cfg_attr(test, allow(clippy::missing_panics_doc))]
```
at the top of `lib.rs`.

Reply:
I still would like this to be done. Is there a reason we can't do this?

Reply:
done in this [commit](https://github.com/spaceandtimelabs/sxt-proof-of-sql/pull/188/commits/af0941d24812df74eed828087e098f2d203b4002)